base_layer/key_manager/Cargo.toml

```diff
@@ -42,5 +39,5 @@ wasm-bindgen-test = "0.3.28"
 
 [features]
 avx2 = ["tari_crypto/simd_backend"]
-js = ["getrandom/js", "js-sys"]
```

Comment:
Might be needed to provide random to JS. Guessing machete might not take this into account

Reply:
The wasm tests _should_ fail if this is the case

Reply:
I manually tested this with 'cargo build - - all-features' if the wasm build with this then it is it should be fine. I picked up 2 issues with this

Reply:
The Js feature is not removed, its still there. Like you say its used by the `wasm` feature branch. 
This PR only removes `"getrandom/js"`

Reply:
If you look at the CI, the key manager wasm tests all pass.